README.md

```diff
@@ -1,6 +1,9 @@
 # Problem Specifications
 
-Shared metadata for Exercism exercises.
+Repository for exercises to be used across tracks.  This includes both problem statements available as markdown and test data described below.
```

Comment:
I put each sentence on separate lines to match the specification that we adhere to (https://exercism.org/docs/building/markdown/markdown).

Also prefixed exercises with 'practice' since Exercism also has concept exercises.

```suggestion
Repository for practice exercises to be used across tracks.
This includes both problem statements available as markdown and test data described below.
```

Reply:
Maybe also mention that the test data is in JSON, similar to how the problem statement is in markdown?

Reply:
That's a good call. I updated the suggested change.